src/test/kotlin/no/nav/familie/ef/sak/beregning/OmregningServiceTest.kt

```diff
@@ -384,4 +451,65 @@ internal class OmregningServiceTest : OppslagSpringRunnerTest() {
     private fun readFile(filnavn: String): String {
         return this::class.java.getResource("/omregning/$filnavn")!!.readText()
     }
+
+    private fun lagreTilkjentYtelse(
+        behandlingId: UUID,
+        stønadsår: Int,
+    ): TilkjentYtelse {
+        val tilkjentYtelse = tilkjentYtelse(
+            behandlingId = behandlingId,
+            personIdent = "321",
+            stønadsår = stønadsår,
+            inntekt = 1,
+            beløp = 1,
+        ) // tidligere beløp betyr ikke noe
```

Comment:
Føler kanskje det er en unødvendig kommentar
```suggestion
        )
```